src/modules/handlers/hello.ts

```diff
@@ -0,0 +1,7 @@
+async function hello (ctx) {
```

Comment:
We are using typescript, so the ideia is to type the variables, what is the ctx type ?

Reply:
ctx type is` ctx: { body = {message: string};`[ A Koa Context encapsulates node's request and response objects into a single object which provides many helpful methods.](https://github.com/koajs/koa/blob/master/docs/api/context.md)